docs/current/sailbot_workspace/setup.md

```diff
@@ -269,15 +272,24 @@ The `setup` task clones the repositories defined in `src/polaris.repos` and upda
 
 The `Build All` task builds all the ROS packages.
 
-## 9. Reload the VS Code terminals and window
+## 9. Setup the MongoDB VS Code extension
+
+Connect the [extension](https://www.mongodb.com/products/vs-code){target=_blank} to the running database:
+[Create a Connection for Deployment](https://www.mongodb.com/docs/mongodb-vscode/connect/#create-a-connection-to-a-deployment){target=_blank}
+
+- Use the default methods: "Paste Connection String" and "Open from Overview Page"
+- Our database's connection string is `mongodb://localhost:27017`
+- See the [MongoDB VS Code extension docs](https://www.mongodb.com/docs/mongodb-vscode/){target=_blank} for how
+  to use it to navigate or explore the database
+

```

Comment:
@jahn18 are there any website setup instructions that everyone should do if they want to run it? Is this step required, or can I remove it or put it somewhere else

Reply:
All the setup for the website is already automated by the container, so there shouldn't be any instructions we need to point in particular. 

> Is this step required, or can I remove it or put it somewhere else

This step isn't _required_, but it would be useful to keep. Maybe we can put it somewhere else? I've personally used the extension to verify the data I inserted into the DB.